### PR TITLE
docs(documentation): reference the rxjs-doc repository

### PR DESCRIPTION
**Pull request by JWO719**
[ReactiveX/rxjs-docs#249](https://github.com/ReactiveX/rxjs-docs/issues/249)

hey @benlesh and @ladyleet, like discussed here is the pr to reference to documentation project. If there is another file where I should add a reminder, please let me know :)

### CODE REVIEW

**Comment by ladyleet** [comment]
@jwo719 rocking our socks off! thx! we super appreciate it. we hope you can join us for some of the rxjs learning team meetings soon! :) ping me on slack! <3

**Comment by sumitarora** [comment]
@JWO719 Nicee 💯

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -9,6 +9,7 @@ If your PR is the addition of a new operator, please make sure all these boxes a
 - [ ] The spec file should have a type definition test at the end of the spec to verify type definition for various use cases
 - [ ] The operator must be documented in JSDoc style in the implementation file, including also the PNG marble diagram image
 - [ ] The operator should be listed in `doc/operators.md` in a category of operators
+- [ ] The operator should also be documented in 'rxjs-docs'. Please reference the pr for the new operator in your pr.
```
Comment (benlesh) [inline .github/PULL_REQUEST_TEMPLATE.md]: > ... should also be documented. See [Documentation Guidelines](https://github.com/ReactiveX/rxjs-docs/blob/master/DOCUMENTATION_GUIDELINES.md)

**Comment by ladyleet** [comment]
omg yay! @JWO719 congrats on your first merged PR to the RxJS project! 🎉

**Comment by JWO719** [comment]
Thank you. Definetly not the last one :blush:

**Pull request merged**